napari_allencell_annotator/widgets/files_widget.py

```diff
@@ -47,14 +49,41 @@ def __init__(self, annotator_model: AnnotatorModel):
         self.setCurrentItem(None)
         self._annotator_model = annotator_model
 
-        self._annotator_model.image_changed.connect(
-            lambda: self.setCurrentItem(self.item(self._annotator_model.get_curr_img_index()))
-        )
+        self._annotator_model.image_changed.connect(self._handle_image_changed)
         self._annotator_model.images_shuffled.connect(self._handle_shuffle)
         self._annotator_model.image_set_added.connect(
             lambda: self._handle_shuffle(self._annotator_model.is_images_shuffled())
         )
         self._annotator_model.image_count_changed.connect(self._handle_image_count_change)
+        self.currentItemChanged.connect(self._handle_file_item_changed)
+
+    def _handle_image_changed(self):
+        with QSignalBlocker(self):
+            if self._annotator_model.get_curr_img_index() is not None:
```

Comment:
This will never be true (see model)

Reply:
removed the if-else statement in https://github.com/aics-int/napari-allencell-annotator/pull/60/commits/6fb901d77bd7917e8ad9b21884b5c326ef9490f6